orchestrator/packages/api/src/controllers/__tests__/grading-queue-controller.test.ts

```diff
@@ -0,0 +1,27 @@
+import { defaultGradingJobConfig } from '../../__mocks__/grading-job-config.mock';
+import { createOrUpdateJob, createOrUpdateImmediateJob  } from '../grading-queue-controller';
+import { Request, Response } from 'express';
+import { errorResponse } from '../utils';
+import { validations } from '@codegrade-orca/common';
+
+
+jest.mock('../utils', () => ({
+  ...jest.requireActual('../utils'),
+  errorResponse: jest.fn()
+}));
+
+describe('job creation controller functions', () => {
+  it.each([createOrUpdateJob, createOrUpdateImmediateJob])(
+    'should throw a 400 with errors when %p is given an invalid request',
+    async (controllerFunc) => {
+      const { files: _files, ...rest } = defaultGradingJobConfig;
+      const validator = validations.gradingJobConfig;
+      validator(rest);
+      expect(!!validator.errors?.length).toBeTruthy();
```

Comment:
This line is pretty cryptic, due to operator precedence.  This should probably be `expect(validator.errors).not.toBe(undefined); expect(validator.errors.length).toBe(greater than 0)`, however those are spelled in jest tests...  At least, that's what I think you're saying!  If not, please clarify further.